migrations/migration-9/pin-set.js

```diff
@@ -164,26 +160,38 @@ function storeItems (blockstore, items) {
     const fanoutLinks = []
 
     for (let i = 0; i < DEFAULT_FANOUT; i++) {
-      fanoutLinks.push(new DAGLink('', 1, EMPTY_KEY))
+      fanoutLinks.push({
+        Name: '',
+        Tsize: 1,
+        Hash: EMPTY_KEY
+      })
     }
 
     if (pins.length <= MAX_ITEMS) {
       const nodes = pins
         .map(item => {
           return ({
-            link: new DAGLink('', 1, item.key),
+            link: {
+              Name: '',
+              Tsize: 1,
+              Hash: item.key
+            },
             data: item.data || new Uint8Array()
           })
         })
         // sorting makes any ordering of `pins` produce the same DAGNode
         .sort((a, b) => {
+          //return a.link.Name.localeCompare(b.link.Name)
```

Comment:
```suggestion
```
can this go away? Or will we use the link locale compare instead of uint8array?

Reply:
Ah, I was experimenting and left that in by accident.